changelogs/unreleased/0000000001-typo_fix.yml

```diff
@@ -0,0 +1,13 @@
+---
+description: Fixed two typos in language.rst file of core docs
+issue-nr: N/A
+change-type: patch
+destination-branches: [master]
+sections:
+  bugfix: "Fixed typos in language.rst file"
+  known-issue: "4 byte unicode character are not supported"
```

Comment:
all the sections other then bug fix should be removed (i.e. you have nothing to add to these sections)